ui/src/components/PathList/PathList.tsx

```diff
@@ -53,11 +54,13 @@ export const PathList: React.FC<Props> = ({ changeList }) => {
                     </FormControl>
                 </div>
                 <div className='changeLogList'>
-                    {filteredChanges?.map((change: DataItem, index: number) => <div key={"changeLogItem" + index} className='changeLogItem'>
-                        <PathListItem changeLogItem={change} showChangeType={type} /></div>)
-                    }
+                    {filteredChanges?.map((change: DataItem, index: number) =>
+                        <PathListItem key={index} changeLogItem={change} showChangeType={type} />
```

Comment:
it is not best practice to use index for react element key..
do we have maybe a key or some identifier for each dataItem that we can use instead?

Reply:
i know...but we dont identifier